plugin/trino-hive-hadoop2/conf/hive-tests-config-apache-hive3.sh

```diff
@@ -0,0 +1,2 @@
+export HADOOP_BASE_IMAGE="ghcr.io/trinodb/testing/hive3.1-hive"
```

Comment:
This file is currently unused. Add something like https://github.com/trinodb/trino/blob/d3a6c2a91d8f786d263ac846245b7f8de5b752a6/.github/workflows/ci.yml#L119 and let's have an issue for that.